src/components/Picker/AutoSuggestText.tsx

```diff
@@ -38,26 +38,28 @@ class AutoSuggestText extends React.PureComponent<Props, {}> {
     return (
       <div className={className}>
         {this.props.stateProps ? this.props.stateProps.chipListState.map((input: any) => <Chip icon={input.icon} onIconClick={input.onIconClick} theme={theme} image={{ url: input.image }} removable={true} onRemove={() => this.props.autoSuggestMethods ? this.props.autoSuggestMethods.chipRemove(input) : null} key={input.key}>{input.text}</Chip>) : null}
-        <Autosuggest
-          className={theme.suggestionsContainer}
-          suggestions={this.props.stateProps ? this.props.stateProps.suggestions : null}
-          onSuggestionsFetchRequested={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.onSuggestionsFetchRequested : null}
-          onSuggestionsClearRequested={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.onSuggestionsClearRequested : null}
-          getSuggestionValue={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.getSuggestionValue : null}
-          onSuggestionSelected={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.onSuggestionSelected : null}
-          renderSuggestion={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.renderSuggestion : null}
-          highlightFirstSuggestion={true}
-          inputProps={this.props.stateProps ? this.props.stateProps.inputProps : null}
-          ref={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.storeInputReference : this.refHolder}
-          theme={{
-            ...theme,
-            container: theme.container,
-            suggestion: theme.suggestionItem,
-            suggestionsList: theme.suggestionsList,
+        <div style={{ width: '100%' }}>
```

Comment:
That's not a good practice, it causes a memory leak within renderes.
One way to accomplish your style is to create a class in pure css and style with "className" property to your div.
You could try remove this div and apply your style to  `theme.suggestionsContainer` as we see in line 43, this way we prevent an additional div.
If not work with suggestionsContainer, just create the css class I said =)